doc/fbchkdoc/mkprntoc.bas

```diff
@@ -296,6 +330,10 @@ while( command(i) > "" )
 			cache_dir = web_cache_dir
 		case "-dev+"
 			cache_dir = dev_cache_dir
+		case "-with-tut-pages"
+			opt_with_tut_pages = true
+		case "-with_dev-pages"
```

Comment:
the `_` in `-with_dev-pages` looks like a typo here

Reply:
if I type it wrong consistently, is it still a typo?, lol.  OK, fixed.